sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -154,6 +161,27 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
       settingKey[Map[String, util.JSON]]("Additional option to include in the generated 'package.json'")
 
     /**
+      * List of entry bundles to generate. By default it generates just one bundle
+      * for your main class.
+      *
+      * @group tasks
+      */
+    val webpackEntries: TaskKey[Seq[(String, File)]] =
+      taskKey[Seq[(String, File)]]("Webpack entry names and corresponding scala.js outputs to process with webpack")
+
+    val webpackEntryPoints: TaskKey[Seq[(String, File)]] =
+      taskKey[Seq[(String, File)]]("Webpack entry names and corresponding entrypoint files to process with webpack")
+
+    val webpackLibraries: TaskKey[Seq[(String, File)]] =
+      taskKey[Seq[(String, File)]]("Webpack entry names and corresponding library files after processing")
+    
+    val webpackLoaders: TaskKey[Seq[(String, File)]] =
+      taskKey[Seq[(String, File)]]("Webpack entry names and corresponding loader files after processing")
+
+    val webpackBundles: TaskKey[Seq[(String, File)]] =
+      taskKey[Seq[(String, File)]]("Webpack entry names and corresponding bundle files after processing")
```

Comment:
IME, exposing less is better. We've come a long way from Scala.js 0.6.x to 1.x into removing internals exposed as keys.

It's easy to expose more later if people need it. Removals are increasingly difficult as your plugin stabilizes.